tflite/AbsOptions.py

```diff
@@ -0,0 +1,28 @@
+# automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
All files in this PR (except setup.py) are autogenerated by flatc, the flatbuffer compiler.